common/lib/common-definitions/src/logger.ts

```diff
@@ -120,3 +120,23 @@ export interface ITelemetryLogger extends ITelemetryBaseLogger {
      */
     shipAssert(condition: boolean, event?: ITelemetryErrorEvent): void;
 }
+
+// ////////////////////// fluidError.ts in common-definitions /////////////////////////////
+/**
+ * Can be decl merged to add stuff like packagename, odspErrorResponse, etc
+ * The logger impl can choose to log anything in here as appropriate, but by default it's not
+ */
+export interface ISensitiveDebugData {
+    innerError?: any;
```

Comment:
we should only support error of type error

Reply:
That's an interesting idea... today code that wraps foreign errors (`createContainerError` and `prepareErrorObject` handle `any` for the wrapped error.

`innerError` is not used for every case - only when there's an error from some dependency that we want to wrap.